artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java

```diff
@@ -1983,6 +1983,12 @@ public void forceFailover() throws Exception {
    }
 
    @Override
+   public void forceShutdown() throws Exception {
+      forceFailover();
+      System.exit(0);
```

Comment:
This is indeed what is happening. I originally invoked forceFailover() which calls server.stop(true);  The server prints the message '06:44:24,063 INFO  [org.apache.activemq.artemis.core.server] AMQ221002: Apache ActiveMQ Artemis Message Broker version 1.5.0-SNAPSHOT [2525ae85-650f-11e6-bc19-0242fee3c111] stopped, uptime 23.737 seconds' and continues to run.

I will raise a jira for this then.